clickhouse_test.go

```diff
@@ -27,12 +27,15 @@ var ddls = []string{
 			a8  Array(UInt8),
 			d   Date,
 			t   DateTime,
-			e   Enum8('one'=1, 'two'=2, 'three'=3)
+			e   Enum8('one'=1, 'two'=2, 'three'=3),
+			d32 Decimal32(4),
+			d64 Decimal64(4),
+			d128 Decimal128(4)
```

Comment:
IMHO you can add `dx Decimal(X, Y)` for better test coverage and examples

Reply:
ok, I will add